### PR TITLE
PlatformBlinkIDLFileDAO

### PR DESCRIPTION
**Pull request by mdittmer**
A filtered DAO that excludes Blink IDL files that are not considered a part of the web platform.

### CODE REVIEW

File: lib/org/chromium/webidl/PlatformBlinkIDLFileDAO.js [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+          return this.CONTAINS(this.IDLFile.PATH, str);
+        }.bind(this);
+
+        return this.NOT(
+            this.OR(
+                CONTAINS('/Source/core/mojo/'),
+                CONTAINS('/Source/core/testing/'),
+                this.AND(
+                    CONTAINS('/Source/modules/'),
+                    CONTAINS('/testing/')),
```
Comment (arobins) [inline lib/org/chromium/webidl/PlatformBlinkIDLFileDAO.js]: Are there paths containing '/testing/' that are part of the web platform?

Comment (mdittmer) [reply lib/org/chromium/webidl/PlatformBlinkIDLFileDAO.js]: Looks like no: https://cs.chromium.org/search/?q=file:third_party/WebKit/Source.*/testing/.*%5C.idl$+-file:/Source/core/testing/+-file:/Source/modules.*/testing/&type=cs

Comment (arobins) [reply lib/org/chromium/webidl/PlatformBlinkIDLFileDAO.js]: In that case, you could probably replace this clause and the /core/testing/ one with a single /testing/ one.

**Review by arobins — APPROVED** [review]
lgtm

**Comment by mdittmer** [comment]
Towards #4

**Pull request merged**